### PR TITLE
Dao/mapper impl merged dev

### PR DESCRIPTION
**Pull request by TomassSimko**
This pr merged all with myBatis and I fixed so everything works we can talk on the meeting but go destroy it if its bug free or not haha (more less no) I will work later on the injections but check this first :)

### CODE REVIEW

**Comment by TomassSimko** [comment]
[Resolve merge conflicts with myBatis to the development](https://trello.com/c/lzHQrPwK/74-resolve-merge-conflicts-with-mybatis-to-the-development)

File: src/main/java/com/movie_collection/bll/helpers/CompareSigns.java [hunk 1]
```diff
@@ -1,6 +1,7 @@
 package com.movie_collection.bll.helpers;
 
 public enum CompareSigns {
+
```
Comment (TomassSimko) [inline src/main/java/com/movie_collection/bll/helpers/CompareSigns.java:4]: sorry here I was trying it with abstract but this is better

**Review by patrikvalentiny — APPROVED** [review]
Great job, did a shit ton of work

**Comment by TomassSimko** [comment]
Thanks @patrikvalentiny I just try to jump on the branch since its a lot to review i apologize but it was required in order to make it work :)

**Review by LordStacker — APPROVED** [review]

**Pull request merged**